addons/sms/static/tests/sms_widget_test.js

```diff
@@ -12,10 +14,16 @@ QUnit.module('fields', {
             partner: {
                 fields: {
                     message: {string: "message", type: "text"},
+                    foo: {string: "Foo", type: "char", default: "My little Foo Value", searchable: true, trim: true},
```

Comment:
searchable and trim not necessary (probably not even implemented in mock server)